src/rtcModule/webrtcPrivate.h

```diff
@@ -133,6 +133,11 @@ class Call: public ICall
         kCallDataReasonCancelled    = 0x05  /// outgoing call was cancelled by caller before receiving any answer from the callee
     };
 
+    enum
+    {
+        kRinging = 0x04,
```

Comment:
It shouldn't end with a comma if it's not the last element of the enumeration. Also, if it's a flag, it would be good to refactorize it to `kFlagRinging` :)